pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/rbac/leader_election_role_binding.go

```diff
@@ -43,6 +44,11 @@ func (f *LeaderElectionRoleBinding) SetTemplateDefaults() error {
 const leaderElectionRoleBindingTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
+  labels:
+    app.kubernetes.io/name: leader-election-rolebinding
+    app.kubernetes.io/component: leader-election
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

Reply:
updated in `kustomize/v1` plugin in commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/8a2c1d7c48ec7743828b14a04ae9de765cde30b2